go/test/endtoend/cluster/mysqlctl_process.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 GitHub Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+)
+
+// MysqlctlProcess is a generic handle for a running mysqlctl command .
+// It can be spawned manually
+type MysqlctlProcess struct {
+	Name         string
+	Binary       string
+	LogDirectory string
+	TabletUID    int
```

Comment:
Where will the actual mysql data and log be stored?

Reply:
Mysql data is stored at : `$VTDATAROOT/vt_${tablet_uid}/data`. And mysql log is stored at `$VTDATAROOT/tmp`

Reply:
What happens when tests are run in parallel? Is tablet_uid unique? otherwise, you will have multiple tests using the same data dir.

Reply:
Yes, tablet_uid is unique. The data directory is different for each of tablets.